docs/webhooks/07-Webhooks-Beta.md

```diff
@@ -1,12 +0,0 @@
----
```

Comment:
I kind of lost how to get to this page when we moved over to Stoplight Docs but there is still a need for a hidden "Beta" page where we can document new webhook events / payloads that have not yet been fully released.